src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs

```diff
@@ -415,6 +422,152 @@ private void HandleCodeActionRequest(string id, string paramsJson)
             _sendToClient(JsonRpcHelper.CreateSuccessResult(id, codeActions));
         }
 
+        /// <summary>
+        /// Handles requests to compute semantic requests for the full document or expression.
```

Comment:
>semantic requests [](http://example.com/codeflow?start=40&length=17)

Can we clarify: what is "semantic request"?

Reply:
this is a typo. Thanks for catching it. It should be semantic tokenization request. I will elaborate more on this in this xml doc.